indico/modules/events/editing/client/js/components/ExtensionList.jsx

```diff
@@ -0,0 +1,39 @@
+import _ from 'lodash';
```

Comment:
missing LICENSE header

Reply:
run `python bin/maintenance/update_header.py indico` should add it

Reply:
`update_header` just prints a warning that the header is missing (doesn't actually add it to the file). By the way, this comment has been resolved but no header is present in the file :P